include/cuda/std/detail/libcxx/include/stdexcept

```diff
@@ -229,7 +231,11 @@ void __throw_logic_error(const char*__msg)
     throw logic_error(__msg);
 #else
     ((void)__msg);
-    _CUDA_VSTD::abort();
+    NV_IF_ELSE_TARGET(
+      NV_IS_DEVICE,
+        __trap();,
+        _CUDA_VSTD::abort();
+    )
```

Comment:
Somewhere above you're made a macro for this, though using if/else on `__CUDA_ARCH__` instead of if target. Should we replace that with what's in here, and just use the macro here to avoid repeating this construct?

Reply:
Which macro was this?